src/polymer/css-import-scanner.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as dom5 from 'dom5';
+import {resolve as resolveUrl} from 'url';
+
+import {ScannedImport} from '../ast/ast';
+
+import {HtmlVisitor, ParsedHtmlDocument} from '../html/html-document';
+import {HtmlScanner} from '../html/html-scanner';
+
+const p = dom5.predicates;
+
+const isCssImportNode = p.AND(p.hasTagName('link'), (node) => {
+  const rel = dom5.getAttribute(node, 'rel') || '';
```

Comment:
You know, this would make a nice predicate factory. Also useful for detecting `class`es. Something like:

```typescript
dom5.predicates.hasSpaceSeparatedAttributeValue('rel', 'import');
```